heuristic/repair_operators/random_repair.py

```diff
@@ -0,0 +1,43 @@
+from numpy.random import RandomState
+import numpy as np
+
+from heuristic.classes import Solution
+from heuristic.functions import create_single_customer_route
+
+
+def random_repair(current: Solution, rnd_state: RandomState) -> Solution:
+    """
+    Sequentially inserts a random customer from the unassigned pool in a random
+    feasible location in the Solution.
+
+    Random repair in Hornstra et al. (2020).
+
+    Tries inserting in one location per route, if it is non feasible for every
+    route, creates a new route.
+    """
+    rnd_state.shuffle(current.unassigned)
+
+    while len(current.unassigned) != 0:
+        customer = current.unassigned.pop()
+
+        indices_routes = np.arange(len(current.routes))
```

Comment:
```route_indices = rnd_state.permutation(len(current.routes))``` is more succinct. I generally forget what's available, but that's documented [here](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.random.RandomState.html).